app/roi/page.tsx

```
@@ -0,0 +1,1074 @@
+import { Button } from '@/components/ui/button'
+import { 
+  Calculator, 
+  ChartLine, 
+  Brain, 
+  Expand, 
+  GraduationCap, 
+  Zap, 
+  DollarSign, 
+  Clock, 
+  BarChart3,
+  Infinity,
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)